ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-incubation.yaml

```diff
@@ -75,6 +75,40 @@ tests:
           cpu: 2000m
           memory: 3Gi
     workflow: optional-operators-ci-gcp
+- as: opendatahub-operator-upgrade
+  steps:
+    test:
+    - as: install
+      cli: latest
+      commands: |
+        export OPERATOR_NAMESPACE=openshift-operators
+        export IMG=quay.io/opendatahub/opendatahub-operator:incubation
+        export BUNDLE_IMG=quay.io/opendatahub/opendatahub-operator-bundle:incubation
+        export ODH_RELEASED_VERSION= oc get packagemanifests opendatahub-operator -o jsonpath='{.status.channels[0].entries[0].version}'
+        make deploy-bundle -e IMG=$IMG BUNDLE_IMG=$BUNDLE_IMG OPERATOR_NAMESPACE=$OPERATOR_NAMESPACE VERSION=$ODH_RELEASED_VERSION # install operator with the version before upgrading
```

Comment:
Since you have already set the env variables , make command can be updated to
```
make deploy-bundle
```

Reply:
> Since you have already set the env variables , make command can be updated to
> 
> ```
> make deploy-bundle
> ```

You mean we don't need to pass env variables with `make deploy-bundle` command ?